keyboards/momokai/tap_trio/config.h

```diff
@@ -27,6 +27,19 @@
 #define DIODE_DIRECTION COL2ROW
 
 
+// Tweak how the mouse cursor moves. https://docs.qmk.fm/#/feature_mouse_keys
+// Accelerated mode (default).
+#define MOUSEKEY_DELAY  10
+#define MOUSEKEY_INTERVAL 20
+#define MOUSEKEY_MOVE_DELTA 8
+#define MOUSEKEY_MAX_SPEED  10
+#define MOUSEKEY_TIME_TO_MAX  30
+#define MOUSEKEY_WHEEL_DELAY  10
+#define MOUSEKEY_WHEEL_INTERVAL 80
+#define MOUSEKEY_WHEEL_MAX_SPEED  8
+#define MOUSEKEY_WHEEL_TIME_TO_MAX  40
```

Comment:
These are all the [default values](https://docs.qmk.fm/#/feature_mouse_keys?id=accelerated-mode)...